packages/api/src/cache.ts

```diff
@@ -171,10 +148,11 @@ export default class Cache {
   }
 
   async load() {
-    // If the class was supplied a raw object, just go ahead and bypass the caching system and
-    // return that.
+    // If the class was supplied a raw object we should still validate and make sure that it's
+    // dereferenced in order for everything to function, but we shouldn't worry about saving it
+    // into the cache directory architecture.
     if (typeof this.uri === 'object') {
```

Comment:
We don't, or document, that people use the dynamic `api` with a JSON object (we use this in some tests to avoid going through our caching architecture) but we weren't dereferencing them before. 😬